packages/client/src/views/MyProfile.vue

```diff
@@ -64,6 +67,16 @@ export default {
           description: 'Send me occasional emails about feature releases, surveys, and other updates.',
         },
       ],
+      breadcrumb_items: [
+        {
+          text: 'Home',
```

Comment:
I noticed that clicking on Home changes the URL to `http://localhost:8080/grants#/my-grants` while the ticket suggests that it should go to `http://localhost:8080/#/grants` (which is also where clicking on the logo takes you).